CONTRIBUTING.md

```diff
@@ -193,28 +235,98 @@ Designation basic format as an example:
 * `information@2x.png` (32x32px)
 * ...
 
+
 ### Color code and status
 
-| Color         | Color code (hex)	| Color code (RGB)	| Used for	|
-| :---------    | :----------- 		| :----------- 		| :----------- 		|
-| orange (logo) | ![#f18f01](https://placehold.co/15x15/f18f01/f18f01.png) `#f18f01`		| `rgb(241, 143, 1)`    | Activated (ex: filter)  |
-| blue (logo)   | ![#0e6e8e](https://placehold.co/15x15/0e6e8e/0e6e8e.png) `#0e6e8e`		| `rgb(14, 110, 142)`   |  	|
-| green (logo)  | ![#9ac155](https://placehold.co/15x15/9ac155/9ac155.png) `#9ac155`		| `rgb(154, 193, 85)`   |  	|
-| dark blue     | ![#95a4b3](https://placehold.co/15x15/95a4b3/95a4b3.png) `#95a4b3`   	| `rgb(149, 164, 179)`  | Default color for icons |
-| red         	| ![#d11d1d](https://placehold.co/15x15/d11d1d/d11d1d.png) `#d11d1d`		| `rgb(209, 29, 29)`      | Error, Fault            |
-| yellow       	| ![#ffd817](https://placehold.co/15x15/ffd817/ffd817.png) `#ffd817`		| `rgb(255, 216, 23)`   | Warning                 |
+| Color	| Color code (hex)	| Color code (RGB)	| Used for	|
+| :---------	| :----------- 	| :----------- 	| :----------- 	|
+| orange (logo)	| ![#f18f01](https://placehold.co/15x15/f18f01/f18f01.png) `#f18f01`	| `rgb(241, 143, 1)`	| Activated (ex: filter)	|
+| blue (logo)	| ![#0e6e8e](https://placehold.co/15x15/0e6e8e/0e6e8e.png) `#0e6e8e`	| `rgb(14, 110, 142)`	|							|
+| green (logo)	| ![#9ac155](https://placehold.co/15x15/9ac155/9ac155.png) `#9ac155`	| `rgb(154, 193, 85)`	|							|
+| dark blue		| ![#95a4b3](https://placehold.co/15x15/95a4b3/95a4b3.png) `#95a4b3`	| `rgb(149, 164, 179)`	| Default color for icons	|
+| red			| ![#d11d1d](https://placehold.co/15x15/d11d1d/d11d1d.png) `#d11d1d`	| `rgb(209, 29, 29)`	| Error, Fault				|
+| yellow		| ![#ffd817](https://placehold.co/15x15/ffd817/ffd817.png) `#ffd817`	| `rgb(255, 216, 23)`	| Warning					|
+
+
+## Interactive-Flex-Query Importer
+
+An XML-compliant document is a file that is structured and formatted according to the rules of the eXtensible Markup Language (XML). It consists of a set of tags and attributes that contain information about the structure and content of the document.
+
+A valid XML document must meet the following conditions:
+
+* It must have a root element that contains all other elements.
+* Each element must be properly closed.
+* Attributes must be defined within a start tag and their values must be specified within quotes.
+* All elements must be properly nested, i.e. they must contain each other and not overlap.
+* There must be no comments or processing instructions outside the root element.
```

Comment:
I don't think this belongs into the contributing document. This is a common sense explanation of XML. The contribution guidelines should be concise and short so that a new contributor is able to get the gist in 10 minutes